osler/inventory/forms.py

```diff
@@ -1,5 +1,5 @@
 from . import models
-from django.forms import ModelForm
+from django.forms import ModelForm, Form
```

Comment:
is this used in this file? if not, it's an unused import and we can get rid of it.